packages/ui/src/helpers/auth.ts

```diff
@@ -98,3 +102,32 @@ export const getActorState = (state: AuthMachineState): AuthActorState => {
 export const getActorContext = (state: AuthMachineState): AuthActorContext => {
   return getActorState(state)?.context;
 };
+
+/**
+ * Creates public facing auth helpers that abstracts out xstate implementation
+ * detail. Each framework implementation can export these helpers so that
+ * developers can send events without having to learn internals.
+ *
+ * ```
+ * const [state, send] = useActor(...);
+ * const { submit } = getSendAliases(send);
+ * submit({ username, password})
+ * ```
+ */
+export const getSendAliases = (send: Sender<AuthEvent>) => {
```

Comment:
nit: Small suggestion to update the function name for clarity

```suggestion
export const getSendEventAliases = (send: Sender<AuthEvent>) => {
```

Reply:
Sounds good, adding this in 👍